tests/acceptance/ENFormCest.php

```diff
@@ -277,20 +274,19 @@ public function createAnEnForm( AcceptanceTester $I ) {
 	}
 
 	/**
-	 * Create a WordPress page and add an enblock in it.
+	 * Create a WordPress page and add an enblock(full-width) in it and check the enform on frontend page.
 	 *
 	 * @depends createAnEnForm
 	 * @group enform
 	 * @group engaging-networks
-	 * @skip
 	 *
 	 * @param AcceptanceTester $I
 	 *
 	 * @throws \Codeception\Exception\ModuleException
 	 */
-	public function createAnEnBlock( AcceptanceTester $I ) {
+	public function createAnTestEnBlockFullWidth( AcceptanceTester $I ) {
```

Comment:
Not a huge deal, but I think there's a typo in the function name here? Shouldn't it be `createAndTestEnBlockFullWidth`?